contracts/metagov/VlAuraFishyImplementation.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "./IAuraLocker.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+// Hidden Hand Aura bribe contract
+interface IAuraBribe {
+    function setRewardForwarding(address to) external;
+
+    function getBribe(
+        bytes32 proposal,
+        uint256 proposalDeadline,
+        address token
+    ) external view returns (address bribeToken, uint256 bribeAmount);
+}
+
+/// @author eswak
+contract VlAuraFishyImplementation {
+    address public constant AURA = 0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF;
+    address public constant VLAURA = 0x3Fa73f1E5d8A792C80F426fc8F84FBF7Ce9bBCAC;
+
+    // TODO: Update this before deploying, Fishy.
+    address public constant OWNER = 0x6ef71cA9cD708883E129559F5edBFb9d9D5C6148;
+
+    modifier onlyOwner() {
```

Comment:
Will he ever need to change the owner? What if he loses access or sells this on? Why not just import OZ's ownable library?

Reply:
As discussed, he's happy with the current version